application/src/test/java/org/opentripplanner/routing/algorithm/mapping/StatesToWalkStepsMapperTest.java

```diff
@@ -53,7 +57,7 @@ void stairs() {
       TestStateBuilder.ofWalking().streetEdge().stairsEdge().streetEdge()
     );
     assertEquals(RelativeDirection.DEPART, walkSteps.get(0).getRelativeDirection());
-    assertEquals(RelativeDirection.STAIRS, walkSteps.get(1).getRelativeDirection());
+    assertTrue(walkSteps.get(1).verticalTransportationUse().get() instanceof StairsUse);
```

Comment:
I think you can do something like assertThat(walkSteps.get(1).verticalTransportationUse().get(), instanceOf(StairsUse.class));